src/index.ts

```diff
@@ -0,0 +1,15 @@
+export class Challenge {
+    buzz (numero : number) {
+        const result = new Promise(function(resolve, reject){
+            if(numero % 3 == 0 && numero % 5 == 0 )
+                resolve('fizzbuzz');
```

Comment:
To make sure to exit the function add a `return` before `resolve` and `reject` calls:
`return resolve('fizzbuzz')`